### DISCUSSION TITLE
Resize images by width and height

### DISCUSSION THREAD

**Discussion by leandro-benedet-garcia** [stoatchat/discussions#109] [Feature Suggestions]
Seems like images are only resized if the width exceeds a certain amount. However. Images that are tall (with lots of height) are not resized, and that can cause loss of conversation because suddenly the whole chat screen is pure image.

It would be interesting for images displayed in chat by default be limited in width and also height.

**Comment by brecert** [comment]
Unless I'm missing something, images are already resized and limited by height.

The default `--attatchment-max-height` is `600px` if I'm not mistaken

![https://github.com/revoltchat/revite/blob/4bf3140f22a59e17a12acab5f961351a6b0c9af5/src/components/common/messaging/attachments/Grid.tsx](https://user-images.githubusercontent.com/11599528/133343724-9d24e69b-268c-4127-b4a9-0dee13ba3a22.png)

**Reply by leandro-benedet-garcia** [reply]
If they are, they are not working in firefox, or the images are still too big. Because I was in the test server and someone sent a image, It flooded the entire screen.

**Reply by brecert** [reply]
What version of firefox, and what os?
Also if you're able to get it, what were the original image dimensions?

**Reply by leandro-benedet-garcia** [reply]
Windows 10, firefox 92.0 (latest version at this time)

I didn't get the image dimensions, but it was a tall image. Way more height than width (it was I believe a screenshot of a whole webpage).

**Reply by brecert** [reply]
I'm having trouble reproducing, I'm guessing it may be that the max height feels too big like you said.

This css snippet can be used in the custom css to reduce the max height of uploaded images to only `250px`.

```css
:root {
  --attachment-max-height: 250px;
}
```
